### ISSUE TITLE
Accelerate-IO rgba32OfFloat bug

### ISSUE THREAD

**Issue by cpdurham** [AccelerateHS/accelerate#274]
```
Main> rgba32OfFloat $ (constant (1,1,1,0) :: Exp (Float,Float,Float,Float))
4294967040
Main> packRGBA32 $ constant (255,255,255,0)
16777215
```

I'm pretty sure its just that the implementation of rgba32OfFloat packs in the reverse order of packRGBA32

**Comment by tmcdonell** [comment]
Ah, dammit. Thanks for pointing that out!

**Comment by tmcdonell** [comment]
Fixed in release 0.15.1.0.
